.aiidalab/setup.cfg

```
@@ -0,0 +1,8 @@
+[aiidalab]
+title = OPTIMADE Client
+authors = Casper Welzel Andersen
+description = AiiDAlab app to query for and import structures from OPTIMADE providers (COD, Materials Cloud, NoMaD, Materials Project, OQMD, and more ...).
+external_url = https://www.optimade.org
+documentation_url = https://github.com/CasperWA/voila-optimade-client#readme
+state = stable
+categories = utility
```

```suggestion
categories = utilities
```
It seems [here](https://github.com/aiidalab/aiidalab-registry/blob/master/categories.yaml) that the value is the plural version and not singular. Maybe that should be unified to be plural/singular?